config/initializers/job_configurations.rb

```diff
@@ -1,3 +1,4 @@
+cron_1m = '0/1 * * * *'
```

Comment:
I'm not _super_ familiar with the usual patterns here, but a minute is pretty frequent, would somewhere in the range of 5-15m intervals be sufficient?

Reply:
I'm fine with bumping it to 5m but ideally we'd really like 1m resolution on this data. This isn't an expensive query as far as I can tell since we're reading from stats that have already been generated. According to good job the full round trip including all the rails bits is only 18.5ms. I'll see if I can find any consensus regarding resolution on this type of data. I know the gitlab infra team also is doing something like this so I"ll start there.

Reply:
Yep, 1m is fine, can always dial back!